packages/p2p-v2/src/utils/file.ts

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
i created the same file, but its called file-uploader.ts in my other PR, can we rename this file, or migrate all the functions from it to here so it won't cause conflicts

Reply:
sure. will update mine.